instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/attributes.go

```diff
@@ -48,7 +54,7 @@ func RegionAttr(region string) attribute.KeyValue {
 	return RegionKey.String(region)
 }
 
-// ServiceAttr returns the AWS service attribute.
+// ServiceAttr returns the RPC service attribute.
```

Comment:
I'm not sure this doc should change. Even though the attributes change, mentioning AWS here feels better.